projects/igniteui-angular/src/lib/tabs/tabs-group.component.ts

```diff
@@ -77,17 +172,36 @@ export class IgxTabsGroupComponent implements AfterContentInit, AfterViewChecked
         private _element: ElementRef) {
     }
 
+    /**
+     * @hidden
+     */
     public ngAfterContentInit(): void {
         if (this.tabTemplate) {
             this._tabTemplate = this.tabTemplate.template;
         }
     }
 
+    /**
+     * @hidden
+     */
     public ngAfterViewChecked() {
         this._element.nativeElement.setAttribute('aria-labelledby', `igx-tab-item-${this.index}`);
         this._element.nativeElement.setAttribute('id', `igx-tabs__group-${this.index}`);
     }
 
+    /**
+     * A method that sets the focus on a tab.
+     * @memberOf {@link IgxDialogComponent}
+     *```html
```

Comment:
@ViewChild("MyTab")
    public tab: IgxTabsComponent;

    ngAfterViewInit(): void {
        // select second tab item
        this.tab.groups.toArray()[1].select();
    }